extensions/amp-analytics/0.1/variables.js

```diff
@@ -302,8 +302,17 @@ export class VariableService {
 
 /**
  * @param {!Window} win
+ */
+export function installVariableService(win) {
+  registerServiceBuilder(win,
+                         'amp-analytics-variables',
+                         VariableService);
```

Comment:
Nit: Generally use +4 spaces when *invoking* functions. Align with the open parens (or +4 spaces) in declarations. Anyways, I think this fits on one line.